src/main/java/com/iridium/iridiumskyblock/commands/BiomeCommand.java

```diff
@@ -0,0 +1,52 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumcore.utils.StringUtils;
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumteams.IridiumTeams;
+import com.iridium.iridiumteams.commands.Command;
+
+import com.iridium.iridiumskyblock.gui.BiomeCategoryGUI;
+import com.iridium.iridiumskyblock.gui.BiomeOverviewGUI;
+import org.bukkit.entity.Player;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+
+public class BiomeCommand extends Command<Island, User> {
+
+    public BiomeCommand() {
+        super(Collections.singletonList("biomes"), "Refresh your Island Biome", "%prefix% &7/is biomes <biome>", "");
+    }
+
+    @Override
+    public void execute(User user, String[] args, IridiumTeams<Island, User> iridiumTeams) {
+        Player player = user.getPlayer();
+        if(args.length == 0){
+            player.openInventory(new BiomeOverviewGUI(player.getOpenInventory().getTopInventory(), IridiumSkyblock.getInstance()).getInventory());
+        }else{
+            String[] commandArguments = Arrays.copyOfRange(args, 0, args.length);
+            Optional<String> categoryName = getCategoryName(String.join(" ", commandArguments), IridiumSkyblock.getInstance());
+
+            if (!categoryName.isPresent()) {
+                player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().noBiomeCategory
+                        .replace("%prefix%", iridiumTeams.getConfiguration().prefix)
+                ));
+                return;
+            }
+
+            player.openInventory(new BiomeCategoryGUI<>(categoryName.get(), player.getOpenInventory().getTopInventory(), IridiumSkyblock.getInstance()).getInventory());
```

Comment:
Whats the difference between an overview and a category?

Reply:
It's a shop - categories are pages of categorized biomes and the overview is a menu with the categories listed